vite.main.config.ts

```diff
@@ -22,8 +23,16 @@ export default defineConfig((env) => {
       rollupOptions: {
         external,
       },
+      sourcemap: true,
     },
-    plugins: [pluginHotRestart('restart')],
+    plugins: [
+      pluginHotRestart('restart'),
+      sentryVitePlugin({
+        authToken: process.env.SENTRY_AUTH_TOKEN,
+        org: process.env.SENTRY_ORG,
+        project: process.env.SENTRY_PROJECT,
+      }),
```

Comment:
This plugin uploads the source map to Sentry to help us identify the actual stack trace of an issue